diff --git a/node_modules/react-firebaseui/FirebaseAuth.js b/node_modules/react-firebaseui/FirebaseAuth.js
index e711df0..c4b6ce9 100644
--- a/node_modules/react-firebaseui/FirebaseAuth.js
+++ b/node_modules/react-firebaseui/FirebaseAuth.js
@@ -30,7 +30,7 @@ var FirebaseAuth = function (_React$Component) {
     _this.firebaseAuth = props.firebaseAuth;
     _this.className = props.className;
     _this.uiCallback = props.uiCallback;
-    _this.unregisterAuthObserver = function () {};
+    _this.unregisterAuthObserver = function () { };
     return _this;
   }
 
@@ -39,7 +39,12 @@ var FirebaseAuth = function (_React$Component) {
 
     require('firebaseui/dist/firebaseui.css');
 
-    var firebaseui = require('firebaseui');
+    // Modification inspirée de https://marabesi.com/development/2021/03/15/firebase-ui-localization.html
+    // Fichier npm buildé à partir des instructions suivantes : https://github.com/firebase/firebaseui-web#building-firebaseui
+    const firebaseResolver = language => require('../../third-party/wrappers/firebaseui/npm__' + language);
+
+    const firebaseui = firebaseResolver('fr')
+    // var firebaseui = require('firebaseui');
 
     return firebaseUiDeletion.then(function () {
       _this2.firebaseUiWidget = firebaseui.auth.AuthUI.getInstance() || new firebaseui.auth.AuthUI(_this2.firebaseAuth);
